### PR TITLE
Adding looping backwards

### PR DESCRIPTION
**Pull request by ZackDeRose**
Looping backwards in JS is best performance, as it allows a one of the operations to be skipped cuz of low-level things.

On my machine:

``` js
const mil = 1000000;
const arr = Array(mil);

console.time('⏲️');
for (let i = 0; i < mil; i++) { }  
console.timeEnd('⏲️');

console.time('⏲️');
for (const v of arr) {} 
console.timeEnd('⏲️');

console.time('⏲️');
arr.forEach(v => v)
console.timeEnd('⏲️');

console.time('⏲️');
for (let i = mil; i < 0; i--) {}
console.timeEnd('⏲️');
```

result:
``` console
⏲️: 3.208ms
⏲️: 20.858ms
⏲️: 2.357ms
⏲️: 1.861ms
```

### CODE REVIEW

**Comment by ZackDeRose** [comment]
@codediodeio - let me know if this is worth adding/wording changes you'd like to see

**Comment by codediodeio** [comment]
Thank you @ZackDeRose Looks great!

File: hugo/content/snippets/javascript-loops-pro-tips.md [hunk 1]
```diff
@@ -50,6 +51,7 @@ for (const v of arr) {}  // 11.7ms
 
 arr.forEach(v => v) // 2.1ms
 
+for (let i = mil; i > 0; i--) {} // ???
```
Comment (ZackDeRose) [inline hugo/content/snippets/javascript-loops-pro-tips.md]: @codediodeio sorry! should have been more explicit about this here! my times were:
```
⏲️: 3.208ms
⏲️: 20.858ms
⏲️: 2.357ms
⏲️: 1.861ms
```
I'm guessing on your's it'd be ~1.6? 1.7?

Comment (ZackDeRose) [reply hugo/content/snippets/javascript-loops-pro-tips.md]: Since this is merged, gimme a :+1: and I'll submit a new PR

**Pull request merged**